llvm/lib/Target/AArch64/AArch64Features.td

```diff
@@ -210,29 +210,29 @@ def FeatureDotProd : ExtensionWithMArch<"dotprod", "DotProd", "FEAT_DotProd",
   "Enable dot product support", [FeatureNEON]>;
 
 def FeatureMPAM : Extension<"mpam", "MPAM", "FEAT_MPAM",
-  "Enable v8.4-A Memory system Partitioning and Monitoring extension">;
+  "Enable Armv8.4-A Memory system Partitioning and Monitoring extension">;
 
 def FeatureDIT : ExtensionWithMArch<"dit", "DIT", "FEAT_DIT",
-  "Enable v8.4-A Data Independent Timing instructions">;
+  "Enable Armv8.4-A Data Independent Timing instructions">;
 
 def FeatureTRACEV8_4 : Extension<"tracev8.4", "TRACEV8_4", "FEAT_TRF",
-  "Enable v8.4-A Trace extension">;
+  "Enable Armv8.4-A Trace extension">;
 
 def FeatureAM : Extension<"am", "AM", "FEAT_AMUv1",
-  "Enable v8.4-A Activity Monitors extension">;
+  "Enable Armv8.4-A Activity Monitors extension">;
 
 def FeatureSEL2 : Extension<"sel2", "SEL2", "FEAT_SEL2",
-  "Enable v8.4-A Secure Exception Level 2 extension">;
+  "Enable Armv8.4-A Secure Exception Level 2 extension">;
 
 def FeatureTLB_RMI : Extension<"tlb-rmi", "TLB_RMI",
   "FEAT_TLBIOS, FEAT_TLBIRANGE",
-  "Enable v8.4-A TLB Range and Maintenance Instructions">;
+  "Enable Armv8.4-A TLB Range and Maintenance Instructions">;
```

Comment:
"Instructions" vs "instructions"

Reply:
Thanks. There are 3 other occurrences of "Instructions" which I'll de-capitalise.